<!DOCTYPE html>
<html lang="en-au">

  <head>
  <meta charset="utf-8">
  <meta name="robots" content="all,follow">
  <meta name="googlebot" content="index,follow,snippet,archive">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>AASC2020 AGM Webinars</title>
  <meta name="author" content="" />

  
  <meta name="keywords" content="aasc, applied statistics">	
  

  
  <meta name="description" content="Site template made by devcows using blogdown R package via hugo">	
  

  <meta name="generator" content="Hugo 0.80.0" />

  <link href='//fonts.googleapis.com/css?family=Roboto:400,100,100italic,300,300italic,500,700,800' rel='stylesheet' type='text/css'>

  
  <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
  <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">

  
  <link href="/css/animate.css" rel="stylesheet">

  
  
    <link href="/css/style.green.css" rel="stylesheet" id="theme-stylesheet">
  


  
  <link href="/css/custom.css" rel="stylesheet">

  
  
    <!--[if lt IE 9]>
        <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
        <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
  

  
  <link rel="shortcut icon" href="/img/favicon.ico" type="image/x-icon" />
  <link rel="apple-touch-icon" href="/img/apple-touch-icon.png" />
  

  <link href="/css/owl.carousel.css" rel="stylesheet">
  <link href="/css/owl.theme.css" rel="stylesheet">

  <link rel="alternate" href="https://aasc2022.netlify.com/index.xml" type="application/rss+xml" title="Australasian Applied Statistics Conference">

  
  <meta property="og:title" content="AASC2020 AGM Webinars" />
  <meta property="og:type" content="website" />
  <meta property="og:url" content="/" />
  <meta property="og:image" content="img/logo.png" />


</head>


  <body>

    <div id="all">

        <header>

          <div class="navbar-affixed-top" data-spy="affix" data-offset-top="200">

    <div class="navbar navbar-default yamm" role="navigation" id="navbar">
        <div class="container">
            <div class="navbar-header">
                <a class="navbar-brand home" href="https://aasc2022.netlify.com/">
                    
                    AASC2022
                    <span class="sr-only">AASC2020 AGM Webinars - go to homepage</span>
                </a>
                <div class="navbar-buttons">
                    <button type="button" class="navbar-toggle btn-template-main" data-toggle="collapse" data-target="#navigation">
                      <span class="sr-only">Toggle Navigation</span>
                        <i class="fa fa-align-justify"></i>
                    </button>
                </div>
            </div>
            

            <div class="navbar-collapse collapse" id="navigation">
                <ul class="nav navbar-nav navbar-right">
                  
                  <li class="dropdown">
                    
                    <a href="/">Home</a>
                    
                  </li>
                  
                  <li class="dropdown">
                    
                    <a href="/news/">News</a>
                    
                  </li>
                  
                  <li class="dropdown">
                    
                    <a href="/speakers/">Speakers</a>
                    
                  </li>
                  
                  <li class="dropdown">
                    
                    <a href="/abstract/">Abstract Submission</a>
                    
                  </li>
                  
                  <li class="dropdown">
                    
                      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Registration <span class="caret"></span></a>
                    <ul class="dropdown-menu">
                      
                        <li><a href="/registration/">Registration</a></li>
                      
                        <li><a href="/accommodation/">Accommodation</a></li>
                      
                    </ul>
                    
                  </li>
                  
                  <li class="dropdown">
                    
                      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Programme <span class="caret"></span></a>
                    <ul class="dropdown-menu">
                      
                        <li><a href="/programme/">Programme</a></li>
                      
                        <li><a href="/workshops/">Workshops</a></li>
                      
                        <li><a href="/social/">Social Programme</a></li>
                      
                    </ul>
                    
                  </li>
                  
                  <li class="dropdown">
                    
                    <a href="/faq/">FAQ</a>
                    
                  </li>
                  
                  <li class="dropdown">
                    
                    <a href="/contact/">Contact</a>
                    
                  </li>
                  
                </ul>
            </div>
            

            <div class="collapse clearfix" id="search">

                <form class="navbar-form" role="search">
                    <div class="input-group">
                        <input type="text" class="form-control" placeholder="Search">
                        <span class="input-group-btn">

                    <button type="submit" class="btn btn-template-main"><i class="fa fa-search"></i></button>

                </span>
                    </div>
                </form>

            </div>
            

        </div>
    </div>
    

</div>




        </header>

        <div id="heading-breadcrumbs">
    <div class="container">
        <div class="row">
            <div class="col-md-12">
                <h1>AASC2020 AGM Webinars</h1>
            </div>
        </div>
    </div>
</div>


        <div id="content">
            <div class="container">

                <div class="row">

                    

                    <div class="col-md-9" id="blog-post">

                        <p class="text-muted text-uppercase mb-small text-right">December 3, 2020</p>

                        <div id="post-content">
                          <p>The 2020 AASC AGM took place on 3 December 2020. You can find the videos of each presentation below with the abstracts.</p>
<ul>
<li><a href="#abstract1">ASReml and Genomic models</a></li>
<li><a href="#abstract2">Development of Genomic tools to support ASReml-R</a></li>
<li><a href="#abstract3">Genstat development in difficult times</a></li>
<li><a href="#abstract4">CycDesigN Version 7.0 Update</a></li>
</ul>
<h2 id="abstract1">ASReml and Genomic models</h2>
<div style="text-align:center;">
       <iframe width="560" height="315" src="https://www.youtube.com/embed/8zpaV1OLvi0" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
</div>
<h4 id="arthur-r-gilmour"><em>Arthur R Gilmour</em></h4>
<p>It is 50 years since Arthur began working as a biometrician with NSW DPI (1970) having majored in biometry in the 4th year of his B Sc AGr (Sydney). He wrote software for the group as well as analysing variety trials and consulting with research scientists. This became the pattern for his career.</p>
<p>His alliance with Robin Thompson commenced in 1981 when Robin visited Massey University and
continues today. Robin proposed the Average Information algorithm to Arthur in 1991 resulting in the development of ASReml in 1996. ASReml was particularly targetedat the analysis of plant breeding data with spatially correlated residuals and of animal breeding data utilising pedigree relationships. It was immediately adopted by many researchers, its core being soon included in Genstat by Sue Welham and into the ASReml-R procedure by David Butler.</p>
<p>Arthur retired from NSW DPI in 2009 and VSN purchased ASReml from NSW DPI and RR in 2010/12. The University of Wollongong funded Arthur to continue supporting ASReml for 3 years until VSN took up that role in 2014. The contract was revised in 2018/9 to resolve issues arising from the development of Echidna (EchidnaMMS.org) whose commercial use is restricted to ASReml licensees. Echidna has proved a useful platform to trial various improvements to ASReml.</p>
<p>ASReml 4.1 (2014) involved a major upgrade to the user interface with the adoption of a functional syntax for model specification similar to that already developed for ASReml-R.</p>
<p>Since then, the focus has been on improving the speed of ASReml 4.2 when fitting less sparse models. The SPARSE matrix methods as implemented in ASReml were inefficient for less sparse models and it has taken 6 years to make reasonable progress speeding up 10 different blocks of code for these models. Three things have contributed to the speed up: use of parallel processing, use of BLOCK processing rather than row by row processing and more efficient ordering of some operations. Even so, I expect further improvement is possible. Recent benchmarking against ASReml 4.1 for a model including a dense genomic relationship matrix of order 3200 showed 4.2 was 10 times faster than 4.1. The factor varies from 1 to 100 depending on the data set and model.</p>
<p>Modelling enhancements include (Double) Hierarchal Generalized Linear Models and bivariate GLMM (but not for multiple threshold models). ASReml-R 4 is based on ASReml 4.1 and a free student licence is available. An update based on ASReml 4.2 is in preparation by David Butler. ASReml 4.2 is also free for approved students.</p>
<p>I thank Jesus Christ and VSN for their continued support and the opportunity to help explore aspects of this incredible world He created.</p>
<h2 id="abstract2">Development of Genomic tools to support ASReml-R</h2>
<div style="text-align:center;">
       <iframe width="560" height="315" src="https://www.youtube.com/embed/PmJK4yJ80_k" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
</div>
<h4 id="salvador-gezan"><em>Salvador Gezan</em></h4>
<p>ASReml is a mature statistical software for the analysis of complex linear mixed models using Residual Maximum likelihood (REML) that is widely used in plant and animal breeding. ASReml is a batch program that can be executed using operating system level commands. An interactive interface, ASReml-W allows to organize, run commands, and display input and output files in a friendly manner using ASReml in the background.</p>
<p>In contrast, ASReml-R is a package that runs inside the R environment with the same capabilities as ASReml, but it uses all the available statistical computing and graphics that R provides. R language is widely used by the scientific community to perform a range of statistical and mathematical analyses through its thousands of libraries available. This environment has allowed ASReml-R users to exploit all its capabilities by developing analytical pipelines around data and output from ASReml-R.</p>
<p>The increased availability of molecular data for many crops and species has expanded the quantitative genetic analyses and allowed for accelerating selection of outstanding genotypes and greater genetic gains. Here, the use of genomic prediction (GP) and genome-wide association studies (GWAS) have become a critical tool to achieve these benefits. ASReml-R is particularly suited to fit genomic selection models, such as GBLUP and rrBLUP, with the use of its powerful REML and mixed model machinery allowing for flexible and complex model fitting. However, these analyses require some complex preprocessing of molecular, phenotypic and, in some cases, pedigree datasets. In addition, the output from these models often requires post-processing to provide the summarized information breeders require for their programs.</p>
<p>ASRgenomics is an R supporting package developed within VSN International to complement and
facilitate the use of this genomic information within ASReml-R. This package contains efficient and useful R functions that help on pre-processing of data, such as filtering and generating additive and dominant genomic matrices, for GBLUP or ssGBLUP analyses. In addition, it has functions that help to assess the quality of the data to perform genomic studies by pairing pedigree-based against genomicbased relationship matrices using both statistical and graphical output, among other things. All these functions are designed to generate objects to be used directly by ASReml-R, or objects from the model fitting that generate additional information.</p>
<p>An open supporting and independent package such as ASRgenomics will allow user to facilitate and expedite analytical pipelines, but will also allow for fast and better interaction with other scientists from the community to incorporate new genomic developments using the machinery of ASReml successfully.</p>
<h2 id="abstract3">Genstat development in difficult times</h2>
<div style="text-align:center;">
          <iframe width="560" height="315" src="https://www.youtube.com/embed/rwVrWA7Gv9Y" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
</div>
<h4 id="roger-payne"><em>Roger Payne</em></h4>
<p>The AASC conference is always a highlight for us at VSN, and it is a real disappointment that we cannot meet this year. Nevertheless it is good to be able to keep in touch by email, MS Teams and Zoom - and to know that you are keeping safe.</p>
<p>The Genstat team is dispersed over the two hemispheres, and so we are used to collaborating at a distance. Genstat development has continued almost unaffected, and I am glad to have this opportunity to tell you what we have been doing. We are developing Genstat-based apps for future cloud services, which we shall be able to demonstrate when we can next meet together. The main activity, though, has been to complete the 21st Edition of Genstat.</p>
<p>A major focus of the 21st Edition has been to provide new menus and procedures to help determine the appropriate fixed terms to include in a REML analysis. This is a weak point of all REML software (including previous Genstat editions), as the focus has always been more on the randommodel. However, with wider usage of REML outside variety trials, QTL analysis etc, the fixed model is becoming more important. You can now explore the model by adding and dropping terms, just as you would in ordinary regression. You can also fit all subsets, and use statistics such as the Akaike or Schwarz Bayesian Information Coefficients to decide on the best model.</p>
<p>There are new menus and procedures for multi-treatment meta analysis based on summary results
from the individual experiments, for response surfaces and for separation plots. There are also many minor extensions, including the ability to provide BLUPs from ANOVA.</p>
<p>Design is another important development area, and we have new commands to construct augmented
block designs and doubly resolvable row-column designs, which should be very useful in all our core biological areas and especially variety trials.</p>
<p>Other new commands allow you to obtain details of the syntax of commands and the source code of procedures stored in a procedure library. This will be especially useful for the support calls that sometimes occur, where the user still has the library but has lost the original source. The source can now be recovered for updates and modification. These provide the basis for new menus to encourage use of Genstat&rsquo;s commands. You can insert standard sections of code into a text box, for example for loops and if-blocks. There is also a menu where you can enter the options and parameters for a command to use any directive or procedure.</p>
<p>So Genstat&rsquo;s life goes on in these difficult times. Please keep safe until Covid 19 is less of a danger, so that we can meet again in 2022.</p>
<h2 id="abstract4">CycDesigN Version 7.0 Update</h2>
<div style="text-align:center;">
       <iframe width="560" height="315" src="https://www.youtube.com/embed/9a2I5mGd1EA" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe> 
</div>
<h4 id="nye-john-dave-whitaker-and-emplyn-williams"><em>Nye John, Dave Whitaker and Emplyn Williams</em></h4>
<p>CycDesigN version 7.0 (CD7) has extensive features for the design of field and glasshouse experiments, including plant phenotyping. This talk will focus on the substantial new facilities available in CD7, in particular neighbour balance and evenness of distribution (NB &amp; ED) designs as applied to resolvable, non-resolvable, single and multi-location partially replicated designs. A number of examples will be presented.</p>

                        </div>
                        
                        

                    </div>
                    

                    

                    

                    <div class="col-md-3">

                        

                        

<div class="panel panel-default sidebar-menu">

    <div class="panel-heading">
      <h3 class="panel-title">Search</h3>
    </div>

    <div class="panel-body">
        <form action="//google.com/search" method="get" accept-charset="UTF-8" role="search">
            <div class="input-group">
                <input type="search" name="q" class="form-control" placeholder="Search">
                <input type="hidden" name="sitesearch" value="https://aasc2022.netlify.com/">
                <span class="input-group-btn">
                    <button type="submit" class="btn btn-template-main"><i class="fa fa-search"></i></button>
                </span>
            </div>
        </form>
    </div>
</div>







<div class="panel panel-default sidebar-menu">

    <div class="panel-heading">
      <h3 class="panel-title">Categories</h3>
    </div>

    <div class="panel-body">
        <ul class="nav nav-pills nav-stacked">
            
            <li><a href="https://aasc2022.netlify.com/categories/invited">invited (10)</a>
            </li>
            
            <li><a href="https://aasc2022.netlify.com/categories/programme">programme (1)</a>
            </li>
            
            <li><a href="https://aasc2022.netlify.com/categories/workshop">workshop (3)</a>
            </li>
            
        </ul>
    </div>
</div>













                        

                    </div>
                    

                    

                </div>
                

            </div>
            
        </div>
        

        <footer id="footer">
    <div class="container">
            
        

        
        <div class="col-md-4 col-sm-6">

          <h4>Contact</h4>

            <strong>Australasian Applied Statistics Conference</strong>
            
            

            <a href="/contact" class="btn btn-small btn-template-main">Go to contact page</a>

            <hr class="hidden-md hidden-lg hidden-sm">

        </div>
        
        

    </div>
    
</footer>







<div id="copyright">
    <div class="container">
        <div class="col-md-12">
            
            <p class="pull-left">Copyright &copy; 2019 - <script type="text/javascript">
  document.write(new Date().getFullYear());
</script>, AGUAi; all rights reserved.</p>
            
            <p class="pull-right">
              Template by <a href="http://bootstrapious.com/free-templates">Bootstrapious</a>.
              

              Ported to Hugo by <a href="https://github.com/devcows/hugo-universal-theme">DevCows</a>
            </p>
        </div>
    </div>
</div>





    </div>
    

    
<script src="//code.jquery.com/jquery-3.1.1.min.js" integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8=" crossorigin="anonymous"></script>
<script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>

<script src="//cdnjs.cloudflare.com/ajax/libs/jquery-cookie/1.4.1/jquery.cookie.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/waypoints/4.0.1/jquery.waypoints.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/Counter-Up/1.0/jquery.counterup.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/jquery-parallax/1.1.3/jquery-parallax.js"></script>

<script src="//maps.googleapis.com/maps/api/js?key=AIzaSyCFhtWLJcE30xOAjcbSFi-0fnoVmQZPb1Y&v=3.exp"></script>

<script src="https://aasc2022.netlify.com/js/hpneo.gmaps.js"></script>
<script src="https://aasc2022.netlify.com/js/gmaps.init.js"></script>
<script src="https://aasc2022.netlify.com/js/front.js"></script>


<script src="https://aasc2022.netlify.com/js/owl.carousel.min.js"></script>

<script>
$('#move-owl').owlCarousel({
    items:2,
    autoPlay: 5000,
    autoplayHoverPause:true
});
</script>


  </body>
</html>
